__tests__/components/editor/PropertyTemplateOutline.test.js

```diff
@@ -277,39 +275,41 @@ describe('<PropertyTemplateOutline /> with propertyTemplate Refs', () => {
       }
     }
   }
-
   const wrapper = mount(<PropertyTemplateOutline {...property}
                                                  handleCollapsed={mockHandleCollapsed}
                                                  handleAddClick={mockHandleAddClick}
                                                  handleMintUri={mockHandleMintUri}
                                                  initNewResourceTemplate={mockInitNewResourceTemplate} />)
-
-  const childOutlineHeader = wrapper.find(OutlineHeader)
   const event = { preventDefault() {} }
 
   // Stub `Config.spoofSinopiaServer` static getter to force RT to come from server
   jest.spyOn(Config, 'spoofSinopiaServer', 'get').mockReturnValue(false)
 
   it('displays a collapsed OutlineHeader of the propertyTemplate label', () => {
+    const childOutlineHeader = wrapper.find(OutlineHeader)
     expect(childOutlineHeader.props().label).toEqual(property.propertyTemplate.propertyLabel)
     expect(childOutlineHeader.props().collapsed).toBeTruthy()
   })
 
   it('clicking removes collapsed state', async () => {
+    // FIXME: this test gives false positive
```

Comment:
What do you think about just removing these tests completely and making issues to write tests (whether they be integration tests or unit tests in different components)? These tests will likely have to be rethought from scratch and the existing ones here are perhaps not at all helpful as starting points.